tbtadm/controller.cpp

```diff
@@ -309,8 +318,8 @@ void tbtadm::Controller::run()
     m_out << "Usage: " << opt_devices << sep << opt_peers << sep << opt_topology
           << sep << opt_approve << " [" << opt_once_flag << "] <route-string>"
           << sep << opt_approve_all << " [" << opt_once_flag << ']' << sep
-          << opt_acl << sep << opt_remove << " <uuid>|<route-string>" << sep
-          << opt_remove_all << "\n";
+          << opt_acl << sep << opt_add << " <route-string>" << sep << opt_remove
+          << " <uuid>|<route-string>" << sep << opt_remove_all << "\n";
     throw std::runtime_error("Wrong usage");
```

Comment:
This string somehow became unreadable as a code and also output is not user-friendly. We should have maybe something like boltctl has:
```
./boltctl -h
Usage:
  boltctl [OPTION…] [COMMAND]

Commands:
  authorize        Authorize a device
  enroll           Authorize and store a device in the database
  forget           Remove a stored device from the database
  info             Show information about a device
  list             List connected and stored devices
  monitor          Listen and print changes

Help Options:
  -h, --help       Show help options
```

Reply:
Yes, moving to Boost.ProgramOption should help ;-)
Any other suggestion how to implement it?

Reply:
I really believe that we should not add any new dependency. We can use similar method as many other Linux tool uses: simple table like in boltctl:
... 
 {"authorize",    authorize,     "Authorize a device"},
 {"enroll",       enroll,        "Authorize and store a device in the database"},
...

BTW, usually those options library handles short and long options and we have only "--once"

Reply:
This PR is OK, no objection to that.

The other side of adding extra unneeded dependency are requirements on the system, running the tool. As you noticed in travis configuration we shall install those extra libraries (not speaking about very super new compiler) just to build the tool.
For the desktop we do not care about the size of the tool itself but sometimes we want to run Linux on very small footprint.